src/Content/PlainTextContentStorageHandler.php

```diff
@@ -99,42 +94,36 @@ protected function contentFileForPage(Page $model, VersionId $versionId, string
 			$directory = $this->model->root();
 		}
 
-		return $directory . '/' . $this->contentFilename($model->intendedTemplate()->name(), $lang);
+		return $directory . '/' . $this->contentFilename($model->intendedTemplate()->name(), $language);
 	}
 
 	/**
 	 * Returns the absolute path to the content file of a site model
-	 *
-	 * @param string $lang Code `'default'` in a single-lang installation
 	 */
-	protected function contentFileForSite(Site $model, VersionId $versionId, string $lang): string
+	protected function contentFileForSite(Site $model, VersionId $versionId, Language $language): string
 	{
-		return $this->contentDirectory($versionId) . '/' . $this->contentFilename('site', $lang);
+		return $this->contentDirectory($versionId) . '/' . $this->contentFilename('site', $language);
 	}
 
 	/**
 	 * Returns the absolute path to the content file of a user model
-	 *
-	 * @param string $lang Code `'default'` in a single-lang installation
 	 */
-	protected function contentFileForUser(User $model, VersionId $versionId, string $lang): string
+	protected function contentFileForUser(User $model, VersionId $versionId, Language $language): string
 	{
-		return $this->contentDirectory($versionId) . '/' . $this->contentFilename('user', $lang);
+		return $this->contentDirectory($versionId) . '/' . $this->contentFilename('user', $language);
 	}
 
 	/**
 	 * Creates a filename with extension and optional language code
 	 * in a multi-language installation
-	 *
-	 * @param string $lang Code `'default'` in a single-lang installation
 	 */
-	protected function contentFilename(string $name, string $lang): string
+	public function contentFilename(string $name, Language $language): string
```

Comment:
Looks like this became public again by rebase or something. If we need it, it should be internal like `contentFile()`.

```suggestion
	protected function contentFilename(string $name, Language $language): string
```

Reply:
Yep, it slipped through when I rebased the PRs